src/Microsoft.AspNetCore.OData/Extensions/ODataServiceCollectionExtensions.cs

```diff
@@ -139,5 +139,38 @@ public static IServiceCollection AddODataQueryFilter(this IServiceCollection ser
             services.TryAddEnumerable(ServiceDescriptor.Singleton<IFilterProvider>(new QueryFilterProvider(queryFilter)));
             return services;
         }
+
+        /// <summary>
+        /// Enables support for case insensitive model binding.
+        /// </summary>
+        /// <param name="services">The services collection.</param>
+        public static IServiceCollection EnableODataCaseInsensitiveModelBinding(this IServiceCollection services)
+        {
+            if (services == null)
+            {
+                throw Error.ArgumentNull("services");
+            }
+
+            services.TryAddEnumerable(ServiceDescriptor.Singleton<IODataModelBinderSettings>(new ODataModelBinderSettings { EnableCaseInsensitiveModelBinding = true }));
```

Comment:
Probably much better for brevity to call the method below with the model

Reply:
#Resolved